pikaur/install_cli.py

```diff
@@ -625,7 +625,7 @@ def ask_about_package_replacements(self) -> None:
                         self.discard_aur_package(installed_pkg_name)
 
     def ask_to_edit_file(self, filename: str, package_build: PackageBuild) -> bool:
-        noedit = self.args.noedit or PikaurConfig().build.get('NoEdit')
+        noedit = not self.args.edit or self.args.noedit or PikaurConfig().build.get('NoEdit')
```

Comment:
shouldn't it be `not self.args.edit and (self.args.noedit or PikaurConfig().build.get('NoEdit'))` ?

Reply:
Got a little confused. Apparently you need parantheses like this ((noedit1 or noedit2) and not edit) for it to work as expected. Pushing new commit now.

Reply:
> Apparently you need parantheses for it to work as expected

could you give some examples please?